ReactiveCocoa/Swift/Property.swift

```diff
@@ -138,46 +136,22 @@ public func <~ <T>(property: MutableProperty<T>, producer: SignalProducer<T, NoE
 	let propertyDisposable = property.producer.start(completed: {
```

Comment:
I can move that inside the closure, but which `Disposable` should the method return if I don't create a `CompositeDisposable`? The `startWithSignal` method doesn't return one, or should I pass the one from the closure into a mutable variable outside of it? Doesn't look that clean either.